cluster/src/main/java/com/hartwig/pipeline/tertiary/lilac/LilacBamSlicer.java

```diff
@@ -135,8 +138,15 @@ public boolean shouldRun(final Arguments arguments) {
     }
 
     private List<BashCommand> buildCommands(final String inputBamFile, final String slicedBamFile) {
-        return List.of(new SambambaCommand("slice", "-L", resourceFiles.hlaRegionBed(), "-o", slicedBamFile, inputBamFile),
-                new SambambaCommand("index", slicedBamFile));
+        return List.of(new SamtoolsCommand("view",
+                "-f",
+                FileTypes.isCram(inputBamFile) ? "cram" : "bam",
+                "-L",
+                resourceFiles.hlaRegionBed(),
+                "-b",
+                "-o",
+                slicedBamFile,
+                inputBamFile), new SambambaCommand("index", slicedBamFile));
```

Comment:
I would very slightly prefer if we called the output `slicedBamOutputFile` or `slicedOutputBamFile` and the input file `inputFile` or `inputAlignedReads` or something as we don't know it's a BAM.